snapshot/restore/local_fs_output.go

```diff
@@ -88,8 +88,8 @@ type FilesystemOutput struct {
 	// SkipTimes when set to true causes restore to skip restoring modification times.
 	SkipTimes bool `json:"skipTimes"`
 
-	// Sparse when set to true causes the restored files to be sparse.
-	Sparse bool `json:"sparse"`
```

Comment:
so this will temporarily break the UI, you'll need to update the field name in separate PR to `htmlui` repo.

Reply:
Gotcha: (https://github.com/kopia/htmlui/pull/76). I wonder if there is a foolproof way to sync these PRs–is `htmlui` imported as a submodule into `kopia`?

Reply:
I already merged HTMLUI PR. Now if you do `go get github.com/kopia/htmluibuild@bf5411c` it should be possible to make an atomic commit that changes both UI and the server.

Reply:
Gotcha, thanks for explaining that. I see you have this commit here https://github.com/kopia/kopia/commit/b8f68df2cf72f3b5249306e988c65fea56752adb doing that. Thanks!